ibm/mas_devops/roles/suite_app_config/vars/predict.yml

```diff
@@ -8,9 +8,10 @@ mas_app_cfg_retries: 30
 
 mas_app_settings_wml_instance_id: "{{ lookup('env', 'WML_INSTANCE_ID') | default('openshift', true) }}"
 mas_app_settings_wml_url: "{{ lookup('env', 'WML_URL') | default('https://internal-nginx-svc.ibm-cpd.svc:12443', true) }}"
-mas_app_settings_wml_version: "{{ lookup('env', 'WML_VERSION') }}"
+mas_app_settings_wml_version: "{{ lookup('env', 'WML_VERSION') | default('4.5', true) }}"
```

Comment:
Can we not set default here ... keep it a required input, unless there is a default that works for all versions of Predict ... put defaults that are suitable for the latest version in the playbooks instead (so they are easy to find).

Reply:
done. changes made as requested.